port/zephyr_port.c

```diff
@@ -117,17 +117,17 @@ esp_loader_error_t loader_port_zephyr_init(const loader_zephyr_config_t *config)
 
 void loader_port_reset_target(void)
 {
-    gpio_pin_set_dt(&enable_spec, false);
+    gpio_pin_set_dt(&enable_spec, CONFIG_SERIAL_FLASHER_RESET_INVERT ? true : false);
     loader_port_delay_ms(CONFIG_SERIAL_FLASHER_RESET_HOLD_TIME_MS);
-    gpio_pin_set_dt(&enable_spec, true);
+    gpio_pin_set_dt(&enable_spec, CONFIG_SERIAL_FLASHER_RESET_INVERT ? false : true);
 }
 
 void loader_port_enter_bootloader(void)
 {
-    gpio_pin_set_dt(&boot_spec, false);
+    gpio_pin_set_dt(&boot_spec, CONFIG_SERIAL_FLASHER_BOOT_INVERT ? true : false);
     loader_port_reset_target();
     loader_port_delay_ms(CONFIG_SERIAL_FLASHER_BOOT_HOLD_TIME_MS);
-    gpio_pin_set_dt(&boot_spec, true);
+    gpio_pin_set_dt(&boot_spec, CONFIG_SERIAL_FLASHER_BOOT_INVERT ? false : true);
```

Comment:
Please remove CONFIG_ prefix here, probably leftover.

Reply:
Zephyr port uses CONFIG_ prefix all around. I don't know the reason, but I assume there is one and I should not go against it.

Reply:
This is true but take a look at how we handle this in the [Zephyr module CMakeLists](https://github.com/espressif/esp-serial-flasher/blob/master/zephyr/CMakeLists.txt).

Reply:
I have to say I don't understand why these new config variables would differ from the old ones in zephyr, but I believe you.